src/odemis/driver/nfpm.py

```diff
@@ -203,15 +205,14 @@ def GetIdentification(self):
              Firmware version (and date)
              serial number
         """
-        resp = self._sendQueryCommand("*IDN")
+        resp = self._sendQueryCommand(b"*IDN").decode("latin1", "backslashreplace")
```

Comment:
You shouldn't use "backslashreplace" for normal bytes->string conversion. This is only used for pretty-printing bytes. In order to actually interpret "resp", you should use `.decode("ascii")`. Move it into the try/except block, so that if there are weird characters, it fails with IOError (and store the decoded string into a separate variable, for instance `resp_str`)